INN/umbrella-mstoday
# Cloudflare LMP fix

## Changes

This pull request makes the following changes:

- Creates a function to add scripts to if they need to be ignored by CloudFlare's Rocket Loader functionality

## Why



For #81

## Testing/Questions

Features that this PR affects:

- Load More Posts script enqueued by Largo


Questions that need to be answered before merging:


Steps to test this PR:

1. View the homepage, verify the LMP functionality still works and nothing else seems broken
2. Find the load more posts script in the page source code and verify it has the `data-cfasync="false"` data attribute
